playbooks/exporter_jmx_install.yml

```
@@ -3,7 +3,7 @@
 
 ---
 - name: jmx-exporter install
-  hosts: hdfs_nn, hdfs_jn, hdfs_dn
+  hosts: hdfs_nn, hdfs_jn, hdfs_dn, yarn_rm, yarn_nm, yarn_ats, mapred_jhs
```

```suggestion
  hosts: hdfs_nn:hdfs_jn:hdfs_dn:yarn_rm:yarn_nm:yarn_ats:mapred_jhs
```

We're using columns when specifying multiple hosts in other playbooks, we should keep the same characeter

---

Yep, done.